test/framework/util.go

```diff
@@ -78,7 +78,7 @@ func ReadFromURI(uri string) (string, error) {
 }
 
 // WaitFor waits for a specific condition to be met
-func WaitFor(namespace, display string, timeout time.Duration, condition func() (bool, error)) error {
+func WaitFor(namespace, display string, timeout time.Duration, condition func() (bool, error), errorCondition func() (bool, error)) error {
```

Comment:
Wouldn't it have more sense to create a different function which will accept the errorCondition and keep this WaitFor as it is?
That way we can avoid having to specify NoErrorCondition() everywhere.

Reply:
I wanted to avoid to have two methods: WaitForWithErrorCondition and another for WaitFor. However, will do as you requested.